src/js/views/globals/prelogin/prelogin.hbs

```diff
@@ -1,20 +1,21 @@
-    <path d="M155.309 11.237l-3.198 10.332c-0.246 0.82-1.039 1.394-1.886 1.394h-0.055c-0.875 0-1.64-0.574-1.886-1.394l-2.652-8.391-2.651 8.391c-0.246 0.82-1.039 1.394-1.913 1.394h-0.055c-0.848 0-1.613-0.574-1.859-1.394l-3.225-10.332c-0.027-0.137-0.055-0.273-0.055-0.41 0-0.684 0.574-1.394 1.449-1.394 0.629 0 1.23 0.41 1.394 1.011l2.515 8.665 2.788-8.528c0.219-0.683 0.848-1.148 1.64-1.148 0.711 0 1.34 0.465 1.585 1.148l2.788 8.528 2.514-8.665c0.164-0.601 0.766-1.011 1.395-1.011 0.847 0 1.449 0.656 1.449 1.394-0.001 0.137-0.028 0.273-0.083 0.41z" fill="#ffffff" />
-    <path d="M168.073 17.141h-8.692c0.191 1.968 1.695 3.635 4.155 3.635 0.956 0 2.049-0.3 2.952-0.875 0.191-0.109 0.41-0.164 0.601-0.164 0.602 0 1.23 0.492 1.23 1.202 0 1.503-3.389 2.187-5.084 2.187-3.964 0-6.915-2.761-6.915-6.943 0-3.827 2.788-6.916 6.697-6.916 3.089 0 5.33 1.859 6.15 4.701 0.164 0.547 0.328 1.175 0.328 1.749 0 0.765-0.629 1.421-1.421 1.421zM163.017 11.62c-2.406 0-3.526 1.913-3.663 3.499h7.326c-0.055-1.53-1.094-3.499-3.663-3.499z" fill="#ffffff" />
-    <path d="M173.129 22.964c-0.82 0-1.449-0.656-1.449-1.449v-15.689c0-0.765 0.629-1.421 1.503-1.421 0.766 0 1.422 0.656 1.422 1.421v15.689c-0 0.793-0.656 1.449-1.477 1.449z" fill="#ffffff" />
-    <path d="M179.060 22.964c-0.82 0-1.449-0.656-1.449-1.449v-15.689c0-0.765 0.629-1.421 1.503-1.421 0.766 0 1.422 0.656 1.422 1.421v15.689c-0 0.793-0.656 1.449-1.477 1.449z" fill="#ffffff" />
-    <path d="M28.781 14.834c-0.117-1.519-0.498-3.012-1.117-4.385-0.618-1.374-1.471-2.627-2.501-3.694-1.030-1.068-2.235-1.949-3.546-2.598-1.311-0.651-2.727-1.067-4.16-1.23-1.432-0.164-2.878-0.081-4.264 0.24-1.385 0.321-2.702 0.883-3.877 1.644-1.175 0.761-2.208 1.719-3.046 2.814-0.838 1.094-1.481 2.325-1.893 3.618-0.414 1.293-0.596 2.648-0.55 3.983l0.263 0.003c0.076-1.305 0.375-2.585 0.879-3.768 0.503-1.183 1.208-2.268 2.067-3.196 0.858-0.929 1.87-1.701 2.971-2.277 1.102-0.576 2.294-0.955 3.505-1.118 1.209-0.164 2.442-0.116 3.617 0.132 1.176 0.247 2.296 0.697 3.299 1.318 1.004 0.62 1.89 1.411 2.614 2.318 0.724 0.907 1.283 1.93 1.649 3.006 0.367 1.076 0.54 2.204 0.521 3.323-0.020 1.122-0.23 2.224-0.617 3.247-0.386 1.024-0.949 1.969-1.648 2.787-0.699 0.818-1.534 1.507-2.449 2.031-0.915 0.525-1.91 0.883-2.925 1.061-1.014 0.178-2.057 0.177-3.060 0.007-1.001-0.169-1.954-0.509-2.815-0.996-0.295-0.167-0.578-0.352-0.85-0.551l4.686-7.132-0.102-0.083-10.27 10.316 0.106 0.099c0.083 0.076 0.156 0.142 0.236 0.211 0.157 0.137 0.318 0.271 0.48 0.401 0.326 0.261 0.662 0.506 1.008 0.737 0.201 0.134 0.405 0.263 0.613 0.386 0.506 0.302 1.029 0.573 1.566 0.81 1.513 0.669 3.135 1.066 4.76 1.18 0.813 0.057 1.622 0.045 2.428-0.035 0.805-0.080 1.603-0.229 2.379-0.445 1.554-0.431 3.020-1.132 4.315-2.052 1.296-0.918 2.423-2.054 3.321-3.338 0.9-1.283 1.572-2.715 1.978-4.21 0.408-1.495 0.548-3.048 0.429-4.565z" fill="#ffffff" />
-  </svg>
+<div class="prelogin">
```

Comment:
This change is just nesting the template in the `prelogin` class. This simplifies how this class is applied